tetris.js

```diff
@@ -277,7 +277,13 @@ module.exports = {
 			}
 
 		}
-		return this.sort(arrayOfCoordinates);
+		return arrayOfCoordinates.sort(function(firstCoordinate,secondCoordinate){
+			let coordinates_X = firstCoordinate.x - secondCoordinate.x;
+			let coordinates_Y = firstCoordinate.y - secondCoordinate.y;
+			return coordinates_X,coordinates_Y;
```

Comment:
Does this work? I thought function should return number

Reply:
not exactly yet,two tests fail
just discovered